_content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Buddhabhasita Dasabhadra Karmamarga Sutra"
+translator: "Upasaka Wong Mou-Lam"
+external_url: "https://ftp.budaedu.org/ebooks/pdf/CE011.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1fB6KVwWegE3IjRD6bbDWNQ7Zfkq6jXyD/view?usp=drive_link"
+course: karma
+tags:
+  - bodhisatta
+  - merit
+publisher: "Buddha Educational Foundation"
+pages: 8
+---
+
+An English translation of the Buddhabhasita Dasabhadra Karmamarga Sutra (Sutra Spoken by Buddha on the Way of Ten Meritorious Deeds). In this sutra, the Buddha enumerates the ten wholesome actions, or meritorious dharmas, which results benefit all beings.
```

Comment:
Just the last sentence. And clean it up a bit:

The Buddha enumerates the ten meritorious deeds and their lavish results to a great Naga king.

Reply:
I might also mention how this sutra builds on EBTs such as AN 10.171.